### ISSUE TITLE
Linker error due to unimplemented functions

### ISSUE THREAD

**Issue by LuizZak** [blend2d/blend2d#17]
Currently a build I'm doing fails because of these unimplemented functions that are defined in public headers:

```
blContextStrokeGlyphRunD
blContextStrokeGlyphRunI
blContextStrokeTextD
blContextStrokeTextI
```

Linker messages:

```
Undefined symbols for architecture x86_64:
  "_blContextStrokeGlyphRunD", referenced from:
      SwiftBlend2D.BLContext.strokeGlyphRun(_: __C.BLGlyphRun, at: __C.BLPoint, font: __C.BLFontCore) -> () in BLContext.o
  "_blContextStrokeGlyphRunI", referenced from:
      SwiftBlend2D.BLContext.strokeGlyphRun(_: __C.BLGlyphRun, at: __C.BLPointI, font: __C.BLFontCore) -> () in BLContext.o
  "_blContextStrokeTextD", referenced from:
      SwiftBlend2D.BLContext.strokeText(_: Swift.String, at: __C.BLPoint, font: __C.BLFontCore) -> () in BLContext.o
  "_blContextStrokeTextI", referenced from:
      SwiftBlend2D.BLContext.strokeText(_: Swift.String, at: __C.BLPointI, font: __C.BLFontCore) -> () in BLContext.o
```

**Comment by kobalicek** [comment]
Thank you! Should be fixed by 9eded845724f8e5ade39b33c8dd41fed81249c93

**Comment by kobalicek** [comment]
I'm closing, I believe this was fixed, reopen if there are more undefined, thanks!
